apps/server/src/app/fhir.controller.ts

```diff
@@ -55,6 +55,36 @@ export class FhirController extends BaseController {
     super(configService, httpService);
   }
 
+
+
+  @Get(':resourceType/:id/([\$])list-ig')
+  @Header('Content-Type', 'text/plain')
+  @HttpCode(200)
+  @ApiOperation({ summary: 'listIg', description: 'List Ig ', operationId: 'listIg' })
+  async listIg(@FhirServerBase() fhirServerBase: string, @Param('resourceType') resourceType: string, @Param('id') id: string, @RequestHeaders('implementationGuideId') contextImplementationGuideId): Promise<boolean> {
+
+    const res = resourceType + "/" + id;
+    const currentOptions: AxiosRequestConfig = {
+      url: buildUrl(fhirServerBase, 'ImplementationGuide', null, null, { resource: res }),
+      method: 'GET'
+    };
+
+    // Get the all the implementation guides for that resource
+    try {
+      const getResponse = await this.httpService.request(currentOptions).toPromise();
+      const bundle:Bundle = getResponse.data;
+      if (bundle.entry.length >= 2 || bundle.entry.length === 1 && contextImplementationGuideId !== bundle.entry[0].resource.id) {
```

Comment:
Looks great!

Reply:
Thanks! I think I would like to change the method name from listIg to validateUniqueIg

Reply:
I agree. That makes sense.